Tools/AP_Periph/can.cpp

```diff
@@ -1708,6 +1709,36 @@ void AP_Periph_FW::can_gps_update(void)
                         &buffer[0],
                         total_size);
     }
+
+    // send the gnss tatus packet
+    {
```

Comment:
I left it at update rate, because that's the rate at which we may have gone unhealthy, or had the status change. It is a single CAN message. We could add more complicated rules to send it faster if it changed anything since last time, but I was adverse to the complexity unless we really thought it merited it.